usdigitalresponse/usdr-gost
# fix: #1386 - Default Sort Validated Desc

Ensure that uploads are automatically sorted by latest validated at the top (#1386)

* added default sorting order to the Uploads page

* added test case for checking the Upload validated at sort dependency

### Ticket #1386 
## Description
Updated the default sort order to sort by Validated At descending.

## Screenshots / Demo Video

## Testing
Ran manual tests as per the below instructions.
Also added a unit test case.

### Automated and Unit Tests

### Manual tests for Reviewer
- Check the page
- Change sorting multiple times
- Click "Reset Filters"
- Check sort
- Click "Group by agency?"
- Check sort
- Click "Reset Filters"
- Click "Only exported to treasury?"
- Check sorting

## Checklist
